src/Provider/Youtube/VideoInfo.php

```diff
@@ -182,6 +182,9 @@ public static function createFromStringWithOptions($string, array $options)
 
         parse_str($string, $video_info);
 
+        // $string may not contain video_id, so we append it manually
+        $video_info['video_id'] = $video_id;
```

Comment:
We should check if video_id isn't set already before we overwrite it.

Reply:
Done inside the new commit